src/iminuit/cost.py

```diff
@@ -1521,7 +1522,7 @@ def __init__(self, n, xe, model, verbose, grad, use_pdf):
                 self._model_xm = self._model_xe[:-1] + 0.5 * dx
         else:
             self._xe_shape = tuple(len(xei) for xei in self.xe)
-            self._model_xe = np.row_stack(
+            self._model_xe = np.vstack(
```

Comment:
Why this replacement?

Reply:
From the old docs, they've always been the same function:

> `np.row_stack` is an alias for [vstack](https://numpy.org/doc/stable/reference/generated/numpy.vstack.html#numpy.vstack). They are the same function.

And they are getting rid of the `row_stack` name in favor of `vstack`:

https://numpy.org/devdocs/numpy_2_0_migration_guide.html#main-namespace

The current one throws a warning. Ruff's NPY check will also flag this.

Reply:
Ok!